tools/localization/regex.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+
+# The regex statements are designed to shortcut so are ordered from most common to least common. The advanced cases will also detect the same result as the simple cases. This is fine.
+def get_localization_regex_list(string):
+  key = re.escape(string)
+  # Regex is ordered from most common to least common
+  return [
+    fr"window\.i18n\('{key}'\)",
```

Comment:
Not brittle at all

Reply:
@Aerilym  we don't need this one, as we have 
```
fr"i18n\('{key}'\)",
fr"i18n\('{key}'(, {{[\S\s.]*}})?\)",
```
just below, and those window.i18n are always going to match the i18n ones

Reply:
This is true but we return early if a regex is matched, so we want to do the more common non-lazy regex search first as its MUCH faster

Reply:
@Aerilym will let you do the testing about this tomorrow

Reply:
@Aerilym reminder

Reply:
I don't think we care. We can keep it as is

Reply:
Cool resolving